src/lib/formats/tzx_cas.cpp

```diff
@@ -736,6 +799,30 @@ static int tzx_cas_do_work( int16_t **buffer )
 			}
 			break;
 
+		case 0x4b:  // TSX Data Block
+			{
+				data_size = get_u32le(&cur_block[1]) - 12;
+				pause_time = get_u16le(&cur_block[5]);
+				pilot = get_u16le(&cur_block[7]);
+				pilot_length = get_u16le(&cur_block[9]);
+				uint16_t zero = get_u16le(&cur_block[11]);
+				uint16_t one = get_u16le(&cur_block[13]);
+				bit0 = (cur_block[15] >> 4) ?: 16;
+				bit1 = (cur_block[15] & 0x0f) ?: 16;
```

Comment:
This is non-standard GNU crap, and hence not portable.

Reply:
https://github.com/mamedev/mame/pull/14206